src/EasyAbp.PrivateMessaging.EntityFrameworkCore/EasyAbp/PrivateMessaging/EntityFrameworkCore/PrivateMessagingDbContextModelCreatingExtensions.cs

```diff
@@ -49,6 +49,7 @@ public static void ConfigurePrivateMessaging(
                 /* Configure more properties here */
                 b.HasIndex(m => m.ToUserId);
                 b.HasIndex(m => m.FromUserId);
+                b.HasIndex(m => m.Category);
```

Comment:
I think the `Category` index doesn't help. You need a compound index instead.

Remove this line and change `b.HasIndex(m => m.ToUserId);` to `b.HasIndex(m => new { m.ToUserId, m.Category });`.